### PR TITLE
added login page

### PR DESCRIPTION
**Pull request by anshikaa12**
>made loginpage
>not yet responsive
login_ecomm

### CODE REVIEW

**Review by prathmeshjagtap — APPROVED** [review]
Looks good, just a suggestion if you can center the username passwords and to inputs, to center it will look nice.

File: css/pages/login/login.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+@import url('/css/layout/main-nav.css');
+@import url('/css/pages/promos.css');
+@import url('/css/pages/featured.css');
+@import url('/css/pages/footer.css');
+@import url('/css/layout/product-nav.css');
+@import url('/css/layout/product-footer.css');
+
+.login-main {
+    padding: 2rem;
+
```
Comment (aryan-13) [inline css/pages/login/login.css:13]: These extra lines can be removed

**Review by aryan-13 — APPROVED** [review]
The code looks good to merge

**Comment by anshikaa12** [comment]
> Looks good, just a suggestion if you can center the username passwords and to inputs, to center it will look nice.

Yes well I tried that too but this looked better!

File: css/pages/login/login.css (hunk 1 above)
Comment (anshikaa12) [reply css/pages/login/login.css:13]: sure

**Pull request merged**